activemodel/lib/active_model/errors.rb

```diff
@@ -613,13 +636,27 @@ def add_from_legacy_details_hash(details)
         }
       end
 
-      def deprecation_removal_warning(method_name)
-        ActiveSupport::Deprecation.warn("ActiveModel::Errors##{method_name} is deprecated and will be removed in Rails 6.2")
+      def deprecation_removal_warning(method_name, alternative_message = nil)
+        message = +"ActiveModel::Errors##{method_name} is deprecated and will be removed in Rails 6.2."
```

Comment:
I think you can remove `+` here 
```suggestion
        message = "ActiveModel::Errors##{method_name} is deprecated and will be removed in Rails 6.2."
```

Reply:
The file is marked as `frozen_string_literal: true` so I have to make this string mutable because I need to append text in the next line.